tests/test_widget.py

```diff
@@ -282,3 +282,12 @@ class TestWidgetRenderingSmallNumber(TestWidgetRenderingBase):
     value = 1e-10
     expected_created = "1e-10"
     expected_db = "1e-10"
+
+
+class TestWidgetRenderingZero(TestWidgetRenderingBase):
+    value = 0
+    expected_created = "0"
+    expected_db = "0.0"
+
+
+# not sure what a db should return if the value is None or ""?
```

Comment:
```suggestion
```
Should not matter if we test for numbers, but good hint.

Please also create a test for `0.0` and `Decimal("0.0"). Thank you :-)

Good Work